test/cli-validator/tests/expected-output.test.js

```diff
@@ -78,6 +78,22 @@ describe('cli tool - test expected output - Swagger 2', function() {
     expect(messageCount).toEqual(ruleCount);
   });
 
+  it('should include the validator version in JSON output', async function() {
+    const program = {};
+    program.args = ['./test/cli-validator/mockFiles/err-and-warn.yaml'];
+    program.default_mode = true;
+    program.json = true;
+
+    await commandLineValidator(program);
+    // exit code is not set for JSON output
```

Comment:
What do mean when you say the exit code isn't set? Any execution of the command line validator should return an exit code and if this doesn't happen (though I'm pretty sure it does in `src/cli-validator/index.js`) then that's a bug that should be fixed. Either way, I think we should check the exit code here

Reply:
I used an adjacent test as a template. This assumption comes from the adjacent test. I will check the exit code and see if we need an issue outlining the exit code problem.

Reply:
Gotcha - and I probably wrote that test haha. Funny how things can seem appropriate in one moment and then not down the road. We do want to make sure the CLI validator _always_ returns a code but like I mentioned, everything is wrapped in the `index.js` file so there shouldn't be any issues. That said, let's check just in case.